metadata/ComputerVision__124100000__metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "changeLog": "Release v2023.10.6.1",
+  "cpu": 0,
+  "description": "Backend server for the AI Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "displayName": "ComputerVision",
+  "gpu": 1,
+  "inputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation. https://docs.uipath.com/activities/other/latest/ui-automation/computer-vision-activities ",
+  "inputType": "JSON",
+  "memory": 0,
+  "mlPackageLanguage": "PYTHON36_CV",
+  "name": "ComputerVision",
+  "outputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation. https://docs.uipath.com/activities/other/latest/ui-automation/computer-vision-activities ",
+  "processorType": "GPU",
+  "projectId": "[project-id]",
+  "retrainable": false,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Computer Vision",
+  "projectDescription": "Backend server for the AI Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "22.10",
+  "languageVersion": 3,
+  "version": 124100000,
```

Comment:
Why is 1 before the model version? If this was discussed before, it's okey, I just want to be sure it's not a typo